😞 looks like we now have a missing dependency, but vendor check says it's ok?

```
09:10:29 integration/service/inspect_test.go:22:2:warning: undeclared name: skip (gosimple)
09:10:29 integration/service/inspect_test.go:15:2:warning: could not import github.com/gotestyourself/gotestyourself/skip (cannot find package "github.com/gotestyourself/gotestyourself/skip" in any of: (gosimple)
```

@dnephin vndr bug?

--- a/libcontainerd/client_windows.go
+++ b/libcontainerd/client_windows.go
@@ -440,9 +440,7 @@ func (clnt *client) AddProcess(ctx context.Context, containerID, processFriendly
 		return -1, err
 	}
 
-	defer func() {
-		container.debugGCS()
-	}()
+	defer container.debugGCS()
 
 	// Note we always tell HCS to
 	// create stdout as it's required regardless of '-i' or '-t' options, so that

--- a/libcontainerd/container_windows.go
+++ b/libcontainerd/container_windows.go
@@ -59,9 +59,7 @@ func (ctr *container) start(attachStdio StdioCallback) error {
 		return err
 	}
 
-	defer func() {
-		ctr.debugGCS()
-	}()
+	defer ctr.debugGCS()
 
 	// Note we always tell HCS to
 	// create stdout as it's required regardless of '-i' or '-t' options, so that

--- a/libcontainerd/utils_windows.go
+++ b/libcontainerd/utils_windows.go
@@ -24,7 +24,7 @@ func (s *LCOWOption) Apply(interface{}) error {
 	return nil
 }
 
-// DebugGCS is a dirty hack for debugging for Linux Utility VMs. It simply
+// debugGCS is a dirty hack for debugging for Linux Utility VMs. It simply
 // runs a bunch of commands inside the UVM, but seriously aides in advanced debugging.
 func (c *container) debugGCS() {
 	if c == nil || c.isWindows || c.hcsContainer == nil {

--- a/vendor.conf
+++ b/vendor.conf
@@ -8,7 +8,7 @@ github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
 github.com/go-check/check 4ed411733c5785b40214c70bce814c3a3a689609 https://github.com/cpuguy83/check.git
 github.com/gorilla/context v1.1
 github.com/gorilla/mux v1.1
-github.com/Microsoft/opengcs v0.3.3
+github.com/Microsoft/opengcs v0.3.4
 github.com/kr/pty 5cf931ef8f
 github.com/mattn/go-shellwords v1.0.3
 github.com/sirupsen/logrus v1.0.1

